todo_subman_community.sh

```diff
@@ -0,0 +1,9 @@
+Subman on debian & ubuntu
```

Comment:
Did you intend to add this file?

Reply:
Nope, mistake :/

Reply:
What I tend to do is type `git status` a lot. Also, `git show` after I've committed to review the diff.

Reply:
The sad truth is that I did exactly what you said and still somehow missed it :/